modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -4087,7 +4067,7 @@ See https://docs.seastar.io/master/[Seastar documentation^]
 
 === raft_timeout_now_timeout_ms
 
-Duration for a timeout now request.
+Timeout for Raft's timeout_now RPC. This RPC is used to force a follower to dispatch a round of votes immediately.
```

Comment:
@Deflaimun I didn't know anything about Raft timeout_now RPC so I googled it but didn't find anything. Does this make sense to you?

Reply:
Yes. According to the developer:

>"timeout_now" is a raft RPC that instructs a follower to dispatch a round of vote right away (which is normally only done if the leader didn't heartbeat for a while). 

If I could draw a correlation to something more simple, it would be similar to a [shutdown in http](https://fullstack.wiki/http/close)